Meadowlark-js/packages/meadowlark-authz-server/src/plugin/NoAuthorizationStorePlugin.ts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { Logger } from '@edfi/meadowlark-core';
+import { CreateAuthorizationClientRequest } from '../message/CreateAuthorizationClientRequest';
+import { CreateAuthorizationClientResult } from '../message/CreateAuthorizationClientResult';
+import { AuthorizationStorePlugin } from './AuthorizationStorePlugin';
+
+export const NoAuthorizationStorePlugin: AuthorizationStorePlugin = {
+  createAuthorizationClient: async (
+    _request: CreateAuthorizationClientRequest,
+  ): Promise<CreateAuthorizationClientResult> => {
+    Logger.warn('NoAuthorizationStorePlugin.createClient(): No authorization store plugin has been configured', null);
```

Comment:
This makes the system inoperable, right? In which case, shouldn't it be an _error_ rather than a _warning_?

Reply:
👍